frontend/src/layouts/footer-layout.tsx

```diff
@@ -3,12 +3,48 @@ function footer() {
   const year = newDate.getFullYear();
 
   return (
-    <footer className="flex min-h-[6vh] flex-col items-center bg-zinc-900 py-2 text-center text-white sm:flex-row sm:justify-between sm:py-3">
-      <section className="text-base sm:pl-4 sm:pl-8 lg:pl-16">WanderLust</section>
-      <section className="text-base sm:pr-16 sm:pr-4 lg:pr-24">
-        <span className="mr-2">&copy;</span>
-        {year} All Rights Reserved
-      </section>
+    <footer className="flex min-h-[6vh] flex-col items-center bg-zinc-900 py-3 pt-10 text-center text-white sm:flex-row sm:justify-center sm:py-3">
+      <div className="mb-8 mt-8">
+        <section className="flex items-center text-xs sm:text-sm">
+          Find an issue with this page?{' '}
+          <span className="pl-1 text-blue-500">
+            <a href="https://github.com/krishnaacharyaa/wanderlust">Fix it on GitHub</a>
+          </span>
+          <span>
+            <a href="https://github.com/krishnaacharyaa/wanderlust">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
```

Comment:
Please extract this out to a different svg file and then import like other svg files in the project

Reply:
Yeah sure